person_capture/video_io.py

```
@@ -3027,6 +3027,104 @@ def grab(self) -> bool:
+        by calling ``grab()`` once for every skipped frame.  For the ffmpeg pipe
+        that still avoided ``retrieve()``, but it crossed Python once per frame
+        and kept all the normal grab/fallback bookkeeping in the hot path.  This
+        method reads whole rawvideo frame blocks from stdout and advances the
+        source-frame counter directly.  It preserves the same sequential decode
+        semantics; it only removes per-frame Python overhead for discarded
+        frames.
+        """
+
+        try:
+            remaining = max(0, int(count))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)